.gitignore

```diff
@@ -12,3 +12,4 @@ data/raw/
 
 # Virtual Environment
 venv*
+src/python-igraph
```

Comment:
Why is that necessary?

Reply:
When I install the requirements it is creating this folder and building igraph in this directory.